src/Surfnet/StepupMiddleware/ApiBundle/Tests/Identity/Projector/AuditLogProjectorTest.php

```diff
@@ -53,7 +54,7 @@ public function auditable_events()
                         new SecondFactorId('ijkl'),
                         new SecondFactorType('sms')
                     )),
-                    BroadwayDateTime::fromString('1970-01-01H00:00:00.000')
+                    BroadwayDateTime::fromString('1970-01-01T00:00:00.000')
```

Comment:
heh, yeah, the tests kept failing because the Stepup DateTime was given a DateTime with the "H" TimeZone. That kept me occupied for a while. 😛 
Stupid PHP for seeing that as a TZ.